src/components/Book.js

```diff
@@ -0,0 +1,23 @@
+const Book = () => {
+  return (
+    <article className='book'>
+      <div>
+        <h5>Action</h5>
+        <h3>The Hunger Games</h3>
+        <p className="book-info">Suzanne Collins</p>
+        <div className='article-footer'>
+          <span>Comments |</span>
+          <span>Remove |</span>
+          <span>Edit |</span>
```

Comment:
- [ ] [REQUIRED] Kindly make these buttons especially the delete button as mentioned in the requirements